src/app/page.tsx

```diff
@@ -5,18 +5,17 @@ import styles from "./page.module.css";
 import InputText from "~/ui/InputText";
 import BtnAdd from "~/ui/BtnAdd";
 
-
 export default function Home() {
   return (
-    <div>
+    <div className="todo-list">
       <h1>To Do List</h1>
-      <div>
+      <div className="todo-list-inputText">
         <InputText onChange={(text) => {
           console.log(text);
         }}/>
       </div>
-      <div>
-        <BtnAdd className="btn-class">Enviar</BtnAdd>
+      <div className="todo-list-btnAdd">
+        <BtnAdd className='container'>Enviar</BtnAdd>
```

Comment:
no need to write `'container'` remove the `className` attr unless you need to add an additional class, remember that the style `container` is already added in the component

```
<button type="button" className={`${style.container} ${className}`}>
```

see that ☝️  the `style.container` is already there, you can make sure the class is being applied in the website

Reply:
👍